README.md

```diff
@@ -208,3 +209,5 @@ a new configuration for it would be appreciated. You can follow these steps:
       [texlab.lua](https://github.com/neovim/nvim-lspconfig/blob/master/lua/lspconfig/texlab.lua).
 
 4. Ask questions on our [Discourse](https://neovim.discourse.group/c/7-category/7) or in the [Neovim Gitter](https://gitter.im/neovim/neovim).
+
+You can also help out by testing [PRs with the `needs-testing`](https://github.com/neovim/nvim-lspconfig/issues?q=is%3Apr+is%3Aopen+label%3Aneeds-testing) label) that affect language servers you use regularly.
```

Comment:
Not sure how worth the labels are given the small number of PRs

Reply:
True, but it has a signal effect as well.